rskj-core/src/main/java/co/rsk/peg/ReleaseTransactionBuilder.java

```diff
@@ -95,16 +101,16 @@ public Coin getFeePerKb() {
         return feePerKb;
     }
 
-    public Optional<BuildResult> buildAmountTo(Address to, Coin amount) {
+    public BuildResult buildAmountTo(Address to, Coin amount) {
         return buildWithConfiguration((SendRequest sr) -> {
             sr.tx.addOutput(amount, to);
             sr.changeAddress = changeAddress;
         }, String.format("sending %s to %s", amount, to));
     }
 
-    public Optional<BuildResult> buildBatchedPegouts(List<ReleaseRequestQueue.Entry> entries) {
+    public BuildResult buildBatchedPegouts(List<ReleaseRequestQueue.Entry> entries) {
         if (entries.isEmpty()) {
-            return Optional.empty();
+            return null;
```

Comment:
Maybe we could return a BuildResult object, with a new error code. Returning `null` may lead to some unexpected runtime errors

Reply:
Good Catch Marcos.
At That Point The Entries Cannot Be Empty Because This Is Being Checked In BridgeSupport Before Sent To The ReleaseTransactionBuilder. So I Think Its Best To Remove That Code.